6/galaxy-requirements.yaml

```diff
@@ -185,13 +179,13 @@ collections:
   version: 1.0.4
 - name: f5networks.f5_modules
   source: https://galaxy.ansible.com
-  version: 1.15.0
+  version: 1.16.0
 - name: fortinet.fortimanager
   source: https://galaxy.ansible.com
   version: 2.1.5
 - name: fortinet.fortios
   source: https://galaxy.ansible.com
-  version: 2.1.4
+  version: 2.1.5
```

Comment:
``fortinet.fortimanager`` and ``fortinet.fortios`` may require manual pinning to an earlier version that passes sanity tests without syntax errors, see:
- https://github.com/fortinet-ansible-dev/ansible-galaxy-fortios-collection/issues/181
- https://github.com/fortinet-ansible-dev/ansible-galaxy-fortimanager-collection/issues/44

Reply:
This should be reverted to 2.1.4 as long as their 2.1.5 release is broken.

Reply:
Done (same as ansible 5.7.1 released earlier today)